### DISCUSSION TITLE
Easy test of the synthesizer (ensure that DAC is working)

### DISCUSSION THREAD

**Discussion by marcel-licence** [marcel-licence/esp32_basic_synth#2] [Q&A]
If you are unsure if the DAC is working modify the setup routine:

setup 
{
[...]

    Synth_NoteOn(64); /* <-- add this command to the end of the routine to ensure one note is still active */
}

Also ensure when using the PCM 5102 that the XMT is pulled up to turn of the mute.

**Comment by Opekusha** [comment]
Thanks for your advice. The sound appeared.
